index.js

```diff
@@ -0,0 +1,151 @@
+
+const URL = 'https://ada-backtrek-api.herokuapp.com/trips';
+
+const loadForm = (data) => {
+  $('#trip-reservation').addClass('bordered');
+  $('#trip-reservation').empty();
+
+
+  $('#trip-reservation').html('<h2>Reserve Trip</h2>');
+  $('#trip-reservation > h2').addClass('bordered-bottom');
+
+  $('#trip-reservation').append(
+    `<form>
+    <input type="hidden" name="id" value="${data.id}">
+    <p>Your Name: <input type="text" name="name"></p>
+    <p>Your Email: <input type="text" name="email"></p>
+    <p>Trip: ${data.name}<br>
+    <input type="submit" value="Reserve"></p>
+    </form>`
+  );
+};
+
+const reportStatus = (message, applyClass = 'success') => {
+  $('#status-message').html(message);
+  $('#status-message').addClass(applyClass);
+};
+
+const reportError = (message, errors) => {
+  let mess = `<p>${message}</p><ul>`
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      mess +=  `<li>${field}: ${problem}<li>`;
+    }
+  }
+  mess += '</ul>';
+  reportStatus(mess, 'err');
+};
+
+
+const showTrips = () => {
+  // style all trips section
+  $('#all-trips').addClass('bordered');
+
+  // style and fill #all-trips header
+  $('#all-trips > h2').html('All Trips');
+  $('#all-trips > h2').addClass('bordered-bottom')
+
+  // style and fill #all-trips ul (#trip-list)
+  $('#trip-list').empty();
+  axios.get(URL)
+
+  .then((response) => {
```

Comment:
Desired style is to include this callback directly below the `get` function call - rather than adding this additional whitespace